CHANGELOG.md

```diff
@@ -39,6 +39,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * `RepoPath` template type now has a `absolute() -> String` method that returns
   the absolute path as a string.
 
+* Added `format_path(path)` template alias that control how file paths are
+  printed in templates.
```

Comment:
The scope is very limited so far (just `jj file list` AFAICT). I think we should say that so users are not confused or disappointed when they try it.

Reply:
https://github.com/jj-vcs/jj/pull/7991 should be good for inspiration. That added a `.absolute()` method. I think we want a `.relative()` (or `.cwd_relative()`?) for converting to a cwd-relative platform-specific string (`/` in Unix, `\` on Windows).

Reply:
I'm sorry, I had simply forgotten that `display()` returns the relative path, so we already have what I was thinking of when I suggested `relative()`. Sorry about the confusion.

So I don't think you're missing anything. As you said, we just need to use the formatter in most places where we currently call `path_converter.format_file_path()`.